### PR TITLE
siyuan-note: Add version 1.4.8

### PR DESCRIPTION
**Pull request by Zuoqiu-Yingyi**
Fix issues.
Click to see detail: [Pull Request #7251 · ScoopInstaller/Extras](https://github.com/ScoopInstaller/Extras/pull/7251)

### CODE REVIEW

File: bucket/siyuan-note.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    "license": {
+        "identifier": "Proprietary",
+        "url": "https://b3log.org/siyuan/eula.html"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/siyuan-note/siyuan/releases/download/v1.4.8/siyuan-1.4.8-win.exe#/siyuan.7z",
+            "hash": "2e72af2c6211595ce4e47c2141e029ce53fb7f934d383f9d8f79493ac6a70b61",
+            "pre_install": [
+                "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\"",
```
Comment (pratikpc) [inline bucket/siyuan-note.json:14]: Could you explain why you need pre-install?

Why would extract_dir not work for example?

Comment (Zuoqiu-Yingyi) [reply bucket/siyuan-note.json:14]: Here `pre-install` works the same as `installer.script`.
Shall I replace it with the code below?
```json
"64bit": {
    "url": "https://github.com/siyuan-note/siyuan/releases/download/v1.4.8/siyuan-1.4.8-win.exe#/dl.7z",
    "hash": "2e72af2c6211595ce4e47c2141e029ce53fb7f934d383f9d8f79493ac6a70b61",
    "installer": {
        "script": [
            "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\"",
            "Remove-Item \"$dir\\`$*\" -Recurse"
        ]
    }
},
"32bit": {
    "url": "https://github.com/siyuan-note/siyuan/releases/download/v1.4.8/siyuan-1.4.8-win32.exe#/dl.7z",
    "hash": "7f97024325040287688ab0ea12ffd8b0e0045680e82c734461254e6e382fd53a",
    "installer": {
        "script": [
            "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-32.7z\" \"$dir\"",
            "Remove-Item \"$dir\\`$*\" -Recurse"
        ]
    }
}
```

Comment (pratikpc) [reply bucket/siyuan-note.json:14]: No I would like to know why we need pre_installer?

Why are the existing approaches not working?

Like a lot of packages have this and am a bit curious

Comment (Zuoqiu-Yingyi) [reply bucket/siyuan-note.json:14]: Actually, I'm confused, too.
I'm just a scoop novick who wants to contribute to a favorite project.😂
These configurations are moved from another niche bucket.👀
[dorado/siyuan.json at master · chawyehsu/dorado](https://github.com/chawyehsu/dorado/blob/master/bucket/siyuan.json)

**Comment by pratikpc** [comment]
@Zuoqiu-Yingyi no directory to persist?

**Comment by Zuoqiu-Yingyi** [comment]
> @Zuoqiu-Yingyi no directory to persist?
No `extract_dir` to persist.
Local tests have been passed.

**Comment by pratikpc** [comment]
Persist is something else

Check https://github.com/ScoopInstaller/Scoop/wiki/Persistent-data

**Comment by Zuoqiu-Yingyi** [comment]
> Persist is something else
> 
> Check https://github.com/ScoopInstaller/Scoop/wiki/Persistent-data

Thank you for your correct.
The user can define the workspace path within the application, so the `persist` does not need to configure.

**Comment by pratikpc** [comment]
@Zuoqiu-Yingyi in that case the manifest works.

I hope one of the maintainers with a better grasp of siyuan would be able to contribute and assist ahead!

**Review by pratikpc — APPROVED** [review]
The manifest is fine.

Someone please verify Siyuan offline once

**Review by rashil2000 — APPROVED** [review]

**Pull request merged**